integration/integration_test.py

```diff
@@ -0,0 +1,44 @@
+"""@package integration_test.py
+Entry point for our integration testing
+"""
+
+import logging
+import os
+import sys
+import unittest
+
+import test_integration_basics
+from common import IpVersion, NighthawkException
+from integration_test_base import IntegrationTestBase
+
+
+def determineIpVersionsFromEnvironment():
+  supported = []
+  versions = os.environ.get("ENVOY_IP_TEST_VERSIONS", "all")
+  if versions == "v4only":
+    supported.append(IpVersion.IPV4)
```

Comment:
Nit: these can all be `supported = ` (and there's no need to empty initialize above, yay Python scoping).

Reply:
done